.github/ISSUE_TEMPLATE/Feature_Request.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Feature Request
+about: Request a new feature
+
+---
+
+<!-- Have a question? Join #thelounge on freenode -->
+
+* *The Lounge version:* 
```

Comment:
I'm not 100% sure the version really matters here, but maybe things such as: "Expected behavior", "Description", etc., everything needed to make a feature request comprehensive.

Also I think it would be useful to have a block comment telling people to check open and closed issues to see if the feature has been requested before, etc.

Reply:
I added a generic "Feature Description" block, I looked at the existing TL issues and I think most feature requests have been going well off without prompting for too much data.